src/eva/plot_tools/dynamic_config.py

```diff
@@ -0,0 +1,132 @@
+# (C) Copyright 2021-2022 NOAA/NWS/EMC
+#
+# (C) Copyright 2021-2022 United States Government as represented by the Administrator of the
+# National Aeronautics and Space Administration. All Rights Reserved.
+#
+# This software is licensed under the terms of the Apache Licence Version 2.0
+# which can be obtained at http://www.apache.org/licenses/LICENSE-2.0.
+
+
+# --------------------------------------------------------------------------------------------------
+
+
+import math
+import numpy as np
+from scipy.stats import skew
+
+from eva.utilities.utils import replace_vars_dict
+
+
+# --------------------------------------------------------------------------------------------------
+
+
+def vminvmaxcmap(logger, option_dict, plots_dict, data_collections):
+
+    # Get percentage of data to use in computing limits. Code will sort the data
+    # by size and then trim the minimum and maximum until this percentage of data
+    # is kept
+    percentage_capture = option_dict.get('percentage capture', 100)
+
+    # Optionally the data might have a channel.
+    channel = option_dict.get('channel', None)
+
+    # Get the data variable to use for determining cbar limits
+    varname = option_dict.get('data variable')
+    varname_cgv = varname.split('::')
+    datavar = data_collections.get_variable_data(varname_cgv[0], varname_cgv[1],
+                                                 varname_cgv[2], channel)
+
+    # Reorder the data array
+    datavar = np.sort(datavar)
+
+    # Decide how many values to throw out on each end of the dataset
+    n = datavar.size
+    n_throw_out = (100-percentage_capture) * n / 200
```

Comment:
Must be Friday PM brain, but why divide by 200? Is this 'half' and going from 1% to 0.01? If so, suggest changing this to:
`n_throw_out = ((1-(percentage_capture/100)) * n ) / 2`
Or scale it after line 28.

Reply:
I changed to line to make the divide by 2 more explicit.